src/serve/create-server-from-route-map.ts

```diff
@@ -36,15 +44,15 @@ export const createServerFromRouteMap = async (routeMap) => {
       const webReq = new Request(`http://localhost${nReq.url}`, {
         headers: nReq.headers,
         method: nReq.method,
-        body: ["GET", "HEAD"].includes(nReq.method) ? undefined : nReq,
+        body: ["GET", "HEAD"].includes(nReq.method ?? "") ? undefined : nReq,
         duplex: "half",
-      })
+      } as any)
```

Comment:
cc @seveibar 

https://github.com/seamapi/edgespec/pull/2/files/1275932962059361a58220860750401913999def#r1421902675

I casted to any here since `Request` and `createServer` types were a bit weird. You think it'd be better if this was a js file with a correlating `d.ts` file instead?

Reply:
bits of any casting is fine, its only problematic if there's a ton of it then the type lives elsewhere, like in the nextlove create-route-spec files where the types live in a separate file and theres a mini type system that eventually gets fully any cast

Reply:
Makes sense, you're saying that when doing type gymnastics, there's a point where doing too much any casting gets to the point where it works against us instead of being helpful so might as well just make a js file and specify a simple type definition. 

Can't relate since i wasn't here for the nextlove implementation, but it makes sense and i'll keep it in mind